app/src/main/java/com/android/sample/ui/listActivities/LikedActivities.kt

```diff
@@ -72,6 +74,8 @@ fun LikedActivitiesScreen(
   val uiState by viewModel.uiState.collectAsState()
   val profile = profileViewModel.userState.collectAsState().value
   val allActivities = (uiState as ListActivitiesViewModel.ActivitiesUiState.Success).activities
```

Comment:
network manager used to check network availability by using ".isNetworkAvailable()" to determine if there is an active and connected network.